src/main/IO/Process/Services/IdentityAccess/RegionPermitRepository.cs

```diff
@@ -81,16 +91,41 @@ public async Task UpdateAsync(RegionPermit regionPermit)
         using var connection = new SqliteConnection(connectionString);
         await connection.OpenAsync();
 
-        var query = $"UPDATE {Constants.TableNames.RegionPermit} SET UserNeuronId = @UserNeuronId, RegionNeuronId = @RegionNeuronId, WriteLevel = @WriteLevel, ReadLevel = @ReadLevel WHERE SequenceId = @SequenceId";
+        var query = $"DELETE FROM {Constants.TableNames.RegionPermit} WHERE SequenceId = @SequenceId";
         using var command = new SqliteCommand(query, connection);
 
+        command.Parameters.AddWithValue("@SequenceId", regionPermit.SequenceId);
+
+        await command.ExecuteNonQueryAsync();
+    }
+
+    public async Task SaveAsync(RegionPermit regionPermit)
+    {
+        AssertionConcern.AssertArgumentNotNull(regionPermit, nameof(regionPermit));
+
+        var id = avatarContextService.Avatar.Id;
+        var connectionString = $@"Data Source=file:{Path.Combine(id, Constants.Databases.IdentityAccessDb)}";
+
+        using var connection = new SqliteConnection(connectionString);
+        await connection.OpenAsync();
+
+        string query;
+        if (regionPermit.SequenceId <= 0)
```

Comment:
When creating regionPermits, the sequenceId is left blank, meaning its value is 0. In that case, SQLite will be the one responsible for the value of the sequenceId since it's auto incremented.